app/assets/javascripts/common/admins.js

```diff
@@ -216,8 +202,18 @@ AdminAccess.prototype = {
     });
   },
 
-  initialize: function () {
-    this.onAsyncLoaded()
+  onDOMLoaded: function() {
+    document.addEventListener('DOMContentLoaded', (_event) => {
+      this.resourceRowCollapseListener()
+    });
+  },
+
+  initialize: function (async = true) {
+    if(async) {
```

Comment:
We need to initialize the collapsable behavior for async and non-async versions of the page.